pages/_app.jsx

```diff
@@ -136,7 +136,7 @@ export default function App(props) {
       <MatomoProvider value={matomoInstance}>
         <StylesProvider jss={jss}>
           <ThemeProvider theme={theme}>
-            <SessionProvider sessionId="pod-browser">
+            <SessionProvider sessionId="pod-browser" restorePreviousSession>
```

Comment:
This flag alone unfortunately isn't enough (otherwise it'd be enabled by default) - you still have to make sure PodBrowser actually navigates back to the page the user was at before the session was revalidated. See https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/restore-session-browser-refresh/#use-session-restore-event-handler

(To reproduce what goes wrong if you don't do this: sign in, navigate to a different page, then refresh. After the session is restored, you will be back at the page you were at when you signed in, rather than where you were when you refreshed.)